packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -186,6 +205,27 @@ const Updates = decorate([
           .sort()
           .map(name => `- ${name}: ${installedPackages[name]}`)
           .join('\n'),
+      xoaReleaseChannelsOptions: (_, { xoaReleaseChannels }) => {
+        const _xoaReleaseChannelsOptions = [...xoaReleaseChannels]
+        return [
+          ...map(_xoaReleaseChannelsOptions, elt => ({
+            label: elt.description,
+            value: elt.id,
+          })),
+          {
+            label: <span className='font-italic'>custom channel</span>,
+            value: 'custom channel',
+          },
+        ]
+      },
+      initialChannel: (_, { xoaReleaseChannels, xoaConfiguration }) => {
+        const channel = find(
+          xoaReleaseChannels,
+          channel => channel.id === xoaConfiguration.channel
+        )
+
+        return (channel && channel.description) || ''
```

Comment:
Shouldn't you return the `id` instead?

Reply:
unfortunetly, fabrice named `channelId` -> `channel`